lib/disabled_cops.yml

```diff
@@ -48,18 +61,10 @@ Style/EachWithObject:
   Enabled: false
 Style/Encoding:
   Enabled: false
-Style/ExtraSpacing:
-  Enabled: false
-Style/FirstArrayElementLineBreak:
-  Enabled: false
-Style/FirstHashElementLineBreak:
-  Enabled: false
-Style/FirstMethodArgumentLineBreak:
-  Enabled: false
-Style/FirstMethodParameterLineBreak:
-  Enabled: false
 Style/FormatString:
   Enabled: false
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
👍  from me on this one being false...I really don't care for it being at the top of every class